graphql_compiler/schema_generation/utils.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+
+
+def toposort_classes(classes):
+    """Sort class metadatas so that a superclass is always before the subclass"""
+    def get_class_topolist(class_name, name_to_class, processed_classes, current_trace):
+        """Return a topologically sorted list of this class's dependencies and class itself
+
+        Args:
+            class_name: string, name of the class to process
+            name_to_class: dict, class_name -> descriptor
+            processed_classes: set of strings, a set of classes that have already been processed
+            current_trace: list of strings, list of classes traversed during the recursion.
+
+        Returns:
+            element of classes list sorted in topological order
```

Comment:
Please try to keep the first word of the `Returns` description as the type description (i.e. `list of .`). It makes it easier for the reader to infer the correct output type at a cursory glance. The same trend and reasoning also applies for `Args`.